tests/ui_tools/test_boxes.py

```diff
@@ -68,6 +68,51 @@ def test_not_calling_send_private_message_without_recipients(self, key,
 
         assert not write_box.model.send_private_message.called
 
+    @pytest.mark.parametrize('key', keys_for_command('SAVE_AS_DRAFT'))
+    @pytest.mark.parametrize('stream_message', [True, False])
+    @pytest.mark.parametrize('draft_message_in_current_session', [
+        True, None,
```

Comment:
You're using a descriptive parameter as a setting here, which is a little confusing.